docs/setting-up.md

```diff
@@ -68,7 +68,7 @@ More information can be found at [this documentation](https://help.github.com/ar
 
 You are encouraged, but not required, to use an IDE to assist many development tasks.
 
-We currently support two IDEs: Eclipse IDE (full support) and IntelliJ IDEA (experimental; limited support).
+We currently support two IDEs: Eclipse IDE (full support) and IntelliJ IDEA.
```

Comment:
Avoid fixing more than one issue in one PR.

Reply:
what is the full code for this issue?